recipes/coconut/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "coconut" %}
+{% set version = "1.2.3" %}
+{% set sha256 = "8965cf8d975eebdf567230aa1598cf8efe72dacfcf7fec853dfd09fed16952d7" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
```

Comment:
It looks like this package uses python entry points. Please add them here:

https://conda.io/docs/building/meta-yaml.html#python-entry-points